### ISSUE TITLE
cannot filter-out tasks at Estimate addin

### ISSUE THREAD

**Issue by ornag123** [cschleiden/azure-boards-estimate#151]
Hi 
I would like to use the DevOps-Estimate. 
when adding stories to estimate, it brings also the tasks. 
we would like to be able to define which object to estimate, and filter-out the tasks. 
how can we do that? 
thanks

![image](https://user-images.githubusercontent.com/75268262/100639191-8908a980-333d-11eb-9e74-30e0befa2f5d.png)

**Comment by Lightras** [comment]
Yes, that's what our team wants too. As a team, we only estimate the story's complexity in points. The task estimations in hours are up to developers when they start working on a story.
So, during our estimate sessions, we have to ignore the tasks, which may sometimes be quite annoying since there are usually multiple tasks under each story.
